<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addrec</class>
 <widget class="QDialog" name="addrec">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/picture/暖黄色背景(1).jpg);</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>460</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;方正小标宋简体&quot;;</string>
   </property>
   <property name="text">
    <string>医生备注</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="visitDateLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>1051</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/picture/透明图.png);
font: 16pt &quot;方正小标宋简体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入记录产生日期，例：2024-05-01</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userIdLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>1051</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/picture/透明图.png);
font: 16pt &quot;方正小标宋简体&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;方正小标宋简体&quot;;</string>
   </property>
   <property name="text">
    <string>记录产生日期</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>1051</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/picture/透明图.png);
font: 16pt &quot;方正小标宋简体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入医疗记录ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>390</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;方正小标宋简体&quot;;</string>
   </property>
   <property name="text">
    <string>诊断结果</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;方正小标宋简体&quot;;</string>
   </property>
   <property name="text">
    <string>治疗</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;方正小标宋简体&quot;;</string>
   </property>
   <property name="text">
    <string>医疗记录ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/picture/图标方.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>520</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;方正舒体&quot;;
border-image: url(:/picture/透明图.png);</string>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>571</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'华文行楷'; font-size:26pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;智慧医疗管家&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;方正小标宋简体&quot;;</string>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="doctorNotesLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>450</y>
     <width>1051</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/picture/透明图.png);
font: 16pt &quot;方正小标宋简体&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入医生备注</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>520</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/picture/透明图.png);
font: 18pt &quot;方正舒体&quot;;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="treatmentLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>1051</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/picture/透明图.png);
font: 16pt &quot;方正小标宋简体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入治疗方式</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="diagnosisResultLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>1051</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/picture/透明图.png);
font: 16pt &quot;方正小标宋简体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入诊断结果</string>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>label_6</zorder>
  <zorder>visitDateLineEdit</zorder>
  <zorder>userIdLineEdit</zorder>
  <zorder>confirmPasswordLineEdit</zorder>
  <zorder>label_5</zorder>
  <zorder>label_7</zorder>
  <zorder>label_9</zorder>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>doctorNotesLineEdit</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>treatmentLineEdit</zorder>
  <zorder>diagnosisResultLineEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
